spec/controllers/api/v1/planets_controller_spec.rb

```diff
@@ -12,13 +12,13 @@
     let!(:first_planet) { Planet.create(
       name: 'Tatooine',
       terrain: 'Desert',
-      url: 'https://en.wikipedia.org/wiki/Tatooine#/media/File:Tatooine_(fictional_desert_planet).jpg',
+      planet_url: 'https://en.wikipedia.org/wiki/Tatooine#/media/File:Tatooine_(fictional_desert_planet).jpg',
       description: 'i dont like sand its coarse and rough',
       ) }
     let!(:second_planet) { Planet.create(
       name: 'Mustafar',
       terrain: 'Lava',
-      url: 'https://vignette.wikia.nocookie.net/starwars/images/d/d1/Hoth.jpg/revision/latest?cb=20071019040631',
+      planet_url: 'https://vignette.wikia.nocookie.net/starwars/images/d/d1/Hoth.jpg/revision/latest?cb=20071019040631',
       description: 'fire burning on the dance floor',
       ) }
```

Comment:
running this spec should break, as you have a trailing comma on line 22. Update accordingly